rust/publish-ghost/src/lib.rs

```diff
@@ -130,7 +130,7 @@ pub struct Cli {
     /// The Ghost domain
     /// 
     /// This is the domain name of your Ghost instance, with an optional port. 
-    #[arg(long, env = "STENCILA_GHOST_DOMAIN", value_parser = parse_host)]
+    #[arg(long, env = "STENCILA_GHOST_DOMAIN", value_parser = parse_host, default_value = "ghost.io")]
```

Comment:
A default of `ghost.io` won't work for anyone, it will always need a subdomain specified. So this isn't a sensible default value to include.

Reply:
Right, my bad.